src/components/FooterIcon/FooterIcon.component.tsx

```diff
@@ -0,0 +1,17 @@
+import type SocialNetwork from '@/interface/interfaceSocialNetwork.ts';
+
+import styles from './footerIcon.module.css';
+
+const SocialNetworkData: React.FC<{ data: SocialNetwork[] }> = ({ data }) => (
+    <ul className={styles.socialNetwork}>
+        {data.map((social, index) => (
+            <li key={index} className={styles.socialItem}>
```

Comment:
if you have completed a task with an asterisk, then you must do it as a separate component that gets its own props

Reply:
Now `FooterIcon` component seems like more `FooterIcons`. Created separate for Icon.

Reply:
and maybe name it `SocialIcon `.